### ISSUE TITLE
"end balance" variable during sim is not consistent with the sim result.

### ISSUE THREAD

**Issue by rohit574** [DeviaVir/zenbot#1685]
### Describe the problem
I just started trying zenbot on osX . used everything default. during running the sim, end balance variable is not consistent with the sim result. in this case end balance should be 452 + 0.0007(BTC converted to USDT) but its showing 953. (screenshot attached).

### Source code / Error logs
<img width="906" alt="screen shot 2018-08-22 at 1 39 34 am" src="https://user-images.githubusercontent.com/3747067/44426738-1250a380-a5ad-11e8-84d4-198801e9d415.png">

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by leozzo** [comment]
Exactly, the balance calculation in sim are completely wrong. After a 12 months sim i made +480% profit, it says me +8600%

Also in real mode are wrong, exchange say X but Zenbot says Y


Inviato da iPhone

**Comment by colinbouvry** [comment]
Hello,
I think I have the same BUG that making sim 
unusable for moment.

Jump mistakes :

![image](https://user-images.githubusercontent.com/985676/50731012-9ff6ef00-115a-11e9-9077-4bfe5da1b44c.png)

end balance: 61095.39529637 (6009.53%)
buy hold: 833.56871076 (-16.65%)
vs. buy hold: 7229.37%
159 trades over 48 days (avg 3.31 trades/day)
win/loss: 28/60
error rate: 68.18%

./zenbot.sh sim gdax.BTC-EUR --strategy pivot --days 45

node 8 zenbot branch unstable 06/01/2019

Any idea ?

**Comment by hathehuyen** [comment]
please fix it @DeviaVir
